src/slo/x_days_sli/x_days_sli_service.py

```diff
@@ -29,15 +31,10 @@ def recalculate_sli(self):
 
     def __should_stay_as_sli_violation(self, table):
         try:
-            return self.filter.exists(self.__create_table_reference(table))
+            if not self.table_existence_filter.exists(SLIViewQuerier.sli_entry_to_table_reference(table)):
```

Comment:
Can you just pass 'table' to the .exists method and make 'sli_entry_to_table_reference' private?

Reply:
done